Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -1068,19 +1068,23 @@
         </ECCustomAttributes>
     </ECEntityClass>
 
-    <ECEntityClass typeName="SurfaceControlVertex" modifier="Sealed" displayLabel="Control Vertex" description="Defines a location point which controls control object.">
+    <ECEntityClass typeName="SurfaceControlVertex" displayLabel="Control Vertex" description="Defines a location point which controls control object.">
         <BaseClass>Vertex</BaseClass>
     </ECEntityClass>
 
-    <ECEntityClass typeName="SurfaceControlVertexRule" modifier="Abstract">
+    <ECEntityClass typeName="SurfaceControlRule" modifier="Abstract" description="Surface control rule.">
```

Comment:
Is this really a SpatialLocationElement? "Rule" class sounds like pure information (no geometry or location)